ui/src/app/styles/colors.ts

```diff
@@ -1,47 +1,28 @@
+import * as Color from 'color';
+
 export default {
-  purple: [
-    '#262262',
-    '#4356a7',
-    '#AB87B3'
-  ],
-  backgroundGrey: '#f1f2f2',
-  red: '#d0021b',
-  green: [
-    '#8bc990',
-    '#7AC79B',
-    '#7CC79B',
-    '#659F3D'
-  ],
-  blue: [
-    '#2691d0',
-    '#5aa6da',
-    '#6CACE4',
-    '#85bde4',
-    '#afd3ed',
-    '#d7eaf6',
-    '#eaf4fb',
-    '#2F2E7E',
-    '#302C71',
-    '#216FB4' // link blue
-  ],
-  gray: [
-    '#4a4a4a',
-    '#6e6e6e',
-    '#929292',
-    '#b7b7b7',
-    '#c3c3c3',
-    '#dbdbdb',
-    '#ededed',
-    '#F1F1F1'
-  ],
+  // Used for buttons and text
+  primary: '#262262',
+  // Used for tab navigation
```

Comment:
The comments are useful – maybe there should be one on each color, succinctly describing its intended use (and current overall color)?

Reply:
Talking to Lou, we decided the other ones were relatively clear semantically.